### PR TITLE
vnext re-generate services

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
@yaohaizh 

Only update on gulpfile.js is needed to enable generate client method without non-required parameters.

I regenerate network as verification. Build pass (now only need minor change on NetworkManagementClientImpl). Let me know whether you need it or not.

**Comment by weidongxu-microsoft** [comment]
Re-generate multiple services, anyway I am testing latest modelerfour.

Storage tests recording included. All pass. (However I should use 2019-06

File: api-specs.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "dir": "azure-mgmt-keyvault",
     "source": "specification/keyvault/resource-manager/readme.md",
     "package": "com.azure.management.keyvault",
-    "args": "--payload-flattening-threshold=1 --tag=package-2018-02"
+    "args": "--payload-flattening-threshold=1 --tag=package-2018-02 --track2-naming=true"
```
Comment (yaohaizh) [inline api-specs.json]: For track 2(vnext) branch, this should be the default option.

Comment (weidongxu-microsoft) [reply api-specs.json]: OK. I will change the option to track1-naming.

**Pull request merged**